### DISCUSSION TITLE
Question about Bot Disk Space

### DISCUSSION THREAD

**Discussion by Movies-Trooper-001** [anasty17/mirror-leech-telegram-bot#1228] [Q&A]
Can anybody explain how bot disk space works I've downloaded more tha 150gb using my bot but yesterday I've mirrored one huge 50gb file to my drive and my bots disk space is got filled and now I can't leech anything. 

May I know why disk got filled when I Mirrored one hug file but not filled after downloading and uploading 150GB's of items ?

How can I clear disk space if it's got filled ? 

And how can I speed up my bot any upgrade needed for good speed in VPS? pls suggest. 

These are the questions. I'm noob in VPS so please excuse me for asking this simple qn. 

My Bot stats 👇
![IMG_20221119_104319](https://user-images.githubusercontent.com/118082697/202835864-e969c96c-5ba6-4fa6-b69f-43381a98d3d9.jpg)

**Comment by anasty17** [comment]
Delete the container check readme
